src/main/java/seedu/address/logic/commands/CreateCommand.java

```diff
@@ -35,7 +35,7 @@ public class CreateCommand extends UndoableCommand {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_NETWORTH + "$1423 "
-            + PREFIX_MEETING_TIME + "15-12-2022 19:00"
+            + PREFIX_MEETING_TIME + "12-15-2022 19:00 "
```

Comment:
Not sure about the change here and why it was made

Reply:
It failed the date time format. and i introduced a space at the end cuz need to leave a space before putting the tag